traitsui/wx/list_str_editor.py

```diff
@@ -586,6 +588,23 @@ def _size_modified(self, event):
         self.control.SetColumnWidth(0, dx - 1)
         event.Skip()
 
+    def _motion(self, event):
+        """Handles the user moving the mouse."""
+        row, _ = self.control.HitTest(wx.Point(event.GetX(), event.GetY()))
+        if row != self._last_row:
+            self._last_row = row
+            if row == -1:
+                tooltip = ""
+            else:
+                tooltip = self.adapter.get_tooltip(
+                    self.object, self.name, row
+                )
+            if tooltip != self._last_tooltip:
```

Comment:
Do we initialize `self._last_tooltip` before the first time this is called?  This looks like a possible bug.

Reply:
This uses the automatic initialization feature of `HasPrivateTraits`. I have clarified it in a comment